.circleci/config.yml

```diff
@@ -45,13 +45,16 @@ commands:
             - build--{{ .Revision }}
 
 jobs:
-  start:
+  start_build:
+    type: no-op
+
+  complete_build:
```

Comment:
This cannot be `no-op` because CircleCI does not report statuses to Github status checks for `no-op`s. We add branch protection against this job.